Documentation/PackageCollections.md

```diff
@@ -0,0 +1,321 @@
+```
+
+To continue user must confirm their trust by passing the `--trust-unsigned` flag:
+
+```bash
+$ swift package-collection add https://www.example.com/packages.json --trust-unsigned
+```
+
+The `--skip-signature-check` flag has no effects on unsigned collections.
+
+### `describe` subcommand
+
+This subcommand shows metadata for a collection or a package included in an imported collection. 
+
+#### Metadata and packages of a collection
+
+`describe` can be used for both collections that have been previously added to the list of the user's configured collections, as well as to preview any other collections.
+
+```bash
+$ swift package-collection describe https://www.example.com/packages.json
+Name: Sample Package Collection
+Source: https://www.example.com/packages.json
+Description: ...
+Keywords: best, packages
+Created At: 2020-05-30 12:33
+Packages:
+    https://github.com/jpsim/yams
+    ...
+```
+
+#### Metadata of a package
+
+`describe` shows the metadata of a package included in an imported collection:
+
+```bash
+$ swift package-collection describe https://github.com/jpsim/yams
+Description: A sweet and swifty YAML parser built on LibYAML.
+Available Versions: 4.0.0, 3.0.0, ...
+Watchers: 14
+Readme: https://github.com/jpsim/Yams/blob/master/README.md
+Authors: @norio-nomura, @jpsim
+--------------------------------------------------------------
+Latest Version: 4.0.0
+Package Name: Yams
+Modules: Yams, CYaml
+Supported Platforms: iOS, macOS, Linux, tvOS, watchOS
+Supported Swift Versions: 5.3, 5.2, 5.1, 5.0
+License: MIT
+```
+
+#### Metadata for a package version
+
+User may view additional metadata for a package version by passing `--version`:
+
+```bash
+$ swift package-collection describe --version 4.0.0 https://github.com/jpsim/yams
+Package Name: Yams
+Version: 4.0.0
+Modules: Yams, CYaml
+Supported Platforms: iOS, macOS, Linux, tvOS, watchOS
+Supported Swift Versions: 5.3, 5.2, 5.1, 5.0
+License: MIT
+```
+
+### `list` subcommand
+
+This subcommand lists all collections that are configured by the user:
+
+```bash
+$ swift package-collection list
+Sample Package Collection - https://example.com/packages.json
+...
+```
+
+### `refresh` subcommand
+
+This subcommand refreshes any cached data manually:
+
+```bash
+$ swift package-collection refresh
+Refreshed 5 configured package collections.
+```
+
+SwiftPM will also automatically refresh data under various conditions, but some queries such as search will rely on locally cached data.
+
+### `remove` subcommand
+
+This subcommand removes as collection from the user's list of configured collections:
+
+```bash
+$ swift package-collection remove https://www.example.com/packages.json
+Removed "Sample Package Collection" from your package collections.
+```
+
+### `search` subcommand
+
+This subcommand searches for packages by keywords or module names within imported collections.
```

Comment:
Is there a difference between "configured" and "imported" (or "added", for that matter) collections? Probably not a huge deal, but sticking to a term might improve understanding.

Reply:
Yes, "configured" means the collection URL is saved the config file; "imported" means the collection is "configured" and its data saved to the database. It's possible to have a collection that is configured but not imported.

Reply:
Ah, great — that makes sense, thanks.